pkgs/by-name/kr/kreya/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+  wrapGAppsHook4,
+  autoPatchelfHook,
+  btar,
+  icu,
+  gtk4,
+  webkitgtk_6_0,
+  openssl,
+}:
+
+let
+  libraries = [
+    icu
+    (lib.getLib stdenv.cc.cc)
+    gtk4
+    webkitgtk_6_0
+    openssl
+  ];
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "kreya";
+  version = "1.18.0";
+
+  src = fetchurl {
+    url = "https://stable-downloads.kreya.app/${finalAttrs.version}/Kreya-linux-x64.tar.gz";
+    hash = "sha256-oXDR1vwU0ikDq4eHFOXTlDgpODH6kVQu/2Ghed9c508=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    autoPatchelfHook
+    wrapGAppsHook4
+  ];
+
+  buildInputs = libraries;
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 kreya -t $out/lib
+    install -Dm755 *.so -t $out/lib
+
+    mkdir -p $out/share/kreya/
+    cp -r example-project $out/share/kreya/
+    cp -r protoc $out/share/kreya/
+    cp kreya.xml $out/share/kreya/
+
+    makeWrapper $out/lib/kreya $out/bin/kreya \
+      --set LD_LIBRARY_PATH "$out/lib:${lib.makeLibraryPath libraries}"
```

Comment:
I am still not really happy if we add all libraries used in buildInputs and by autoPatchelfHook also to here. From my experience we only need GUI libraries like libGL or vulkan here.